modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1155,7 +1155,11 @@ export class Wallet implements IWallet {
       }
       addressParams.forwarderVersion = forwarderVersion;
     } else if (this._wallet.multisigType === 'tss' && this.baseCoin.getMPCAlgorithm() === 'ecdsa') {
-      addressParams.forwarderVersion = 3;
+      if (this._wallet.multisigTypeVersion === 'MPCv2') {
+        addressParams.forwarderVersion = 4;
```

Comment:
Where is the `forwarderVersion` documented? Just want to make sure we are using the correct value here.

Reply:
Not sure where it's documented, but we've already tested it working with `forwarderVersion` = 4

Reply:
@BitGo/ethalt-team  can u confirm the correct forwarder version number for MPCv2 wallets?

Reply:
forwarder version 4 is correct

Reply:
This change is really eth specific, Only eth like coins have forwarder versions. So when it comes to cosmos etc which doesn't use forwarders at all this could cause issues. I think by default, if you don't pass in forwarder version, it should by default create the address with version 4 and it should be the case for all coins where it will create the address according to its defaults if we didn't explicitly specify. If we are able to create other versions of forwarders, Kindly let me know, I can raise a ticket for it

Reply:
Removed the coin specific logic completely as discussed offline and will let WP handle the defaults